assets/css/style.css

```diff
@@ -1,317 +1,317 @@
 @font-face {
-    font-family: "Agustina";
-    src: url(../fonts/Agustina.woff);
```

Comment:
Please avoid indentation and linting changes

Reply:
@smaranjitghose  it happend because of my VS-Code formatter. I resolved the issue by giving anchor tag 's :visited CSS pseudo-class a same **rgba(255, 255, 255, 0.5)** color. Please check the line number 297.

Reply:
@smaranjitghose it was done unintentionally.
I can make a new PR after disabling my code formatter if you want